### ISSUE TITLE
Equals and plus not rendering in equations

### ISSUE THREAD

**Issue by JusteRaimbault** [UCL/ucl-beamer#3]
Using the ucl beamer template (with TeXlive 2017 on Mac OSX 10.12), the signs "=" and "+" are not rendered when in math mode - I have to force them using "\textrm{=}".
Did not do anything about fonts, could it be linked?

**Comment by TobiasKappe** [comment]
Dear Juste,

First, let me apologise for replying almost a year after you filed this issue. It turns out that notification emails from GitHub have been ending up in my spam folder, which made me miss this.

I'm unable to reproduce the issue on my end, with TeXLive 2021 on Linux using the sample file in the repository. Can you please check if the issue is still present on your end? If so, perhaps we can find a solution.

Best,

Tobias

**Comment by lukeg101** [comment]
Dear Juste,
I am also sorry for the delay, I was unable to reproduce this with TexLive (3.14...) on MacOS (12.1) using the sample tex file with the following document (everything below the first slide removed, with just the symbols in question):
```
... preamble in sample.tex
\begin{document}

\begin{frame}
  \frametitle{First Slide}
  \begin{itemize}
  \item $=$
  \item $+$
  \item \(=\)
  \item \(+\)
  \item \begin{math}=\end{math}
  \item \begin{math}+\end{math}
  \end{itemize}
\end{frame}

\end{document}
```

Did you import any packages that adds symbols? What did you use to build your pdf? Or if you did solve this please let us know and we can close this.
Thanks
Luke

**Comment by JusteRaimbault** [comment]
Dear both,

Thanks for your answer.
I do not have access to my Mac anymore, working now on Linux and the template works fine.

Back in the days I was using pdflatex to build, and packages added:
```
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{mathptmx,amsmath,amssymb,graphicx,bibentry,bbm,ragged2e}
\usepackage[english]{babel}
```

so maybe amssymb or bbm clashing with some packages already loaded by the template, or an issue with fontenc?

I think the issue could be closed as it was with an old OSX version and an old Texlive version.
Thanks.

**Comment by TobiasKappe** [comment]
I can reproduce this issue on my machine; it looks like the culprit is `mathptmx`.

That package [appears to be obsolete](https://ctan.org/pkg/mathptmx?lang=en), and [superseded by newtxmath](https://ctan.org/pkg/newtx), which does not cause the same issue.

I propose that we put a note in the README and then close this issue.

**Comment by lukeg101** [comment]
Thanks for that, it looks as though we have a solution! I'll submit a PR for the README next week and link it here in case either of you would like to review :)

**Comment by TobiasKappe** [comment]
Perfect, thanks!

**Comment by lukeg101** [comment]
Hi folks, apologies for my delay [here](https://github.com/UCL/ucl-beamer/pull/4) is the PR :)

**Comment by TobiasKappe** [comment]
Resolved by #4.
